dockertest.go

```diff
@@ -579,7 +579,7 @@ func (d *Pool) CurrentContainer() (*Resource, error) {
 	case *dc.NoSuchContainer:
 		return nil, ErrNotInContainer
 	default:
-		return nil, errors.Wrap(err, "")
+		return nil, fmt.Errorf("%w", err)
```

Comment:
Could be just err returned, this wrapping is not needed

Reply:
Updated. Please take a look again. Thank you.